packages/wonder-blocks-core/util/types.ts

```diff
@@ -215,11 +213,7 @@ export type TextViewSharedProps = {
 
     // Used by tooltip bubble
     "data-placement"?: string,
-
-    ...AriaProps,
-
-    ...EventHandlers,
-};
+} & AriaProps & EventHandlers;
```

Comment:
There's no spread operator for types.  This should be okay in most cases.  The only situation where we'd have to do something different is if we want to replace `x: number` with `x: string`.  Replacing `x: any` with `x: string` is fine as is replacing `x: string` with `x: "foo"`.